osidb/filters.py

```diff
@@ -222,6 +222,10 @@ class Meta:
             + LTE_GTE_LOOKUP_EXPRS
             + DATE_LOOKUP_EXPRS,
             "affects__trackers__errata__advisory_name": ["exact"],
+            "affects__trackers__errata__shipped_dt": ["exact"]
+            + LT_GT_LOOKUP_EXPRS
+            + LTE_GTE_LOOKUP_EXPRS
+            + DATE_LOOKUP_EXPRS,
```

Comment:
I would like to avoid these kind of filters since they are insane, instead what we can do is create a filter simply named `errata_shipped_dt` or similar and link it to a function, similar to what we do with `changed_after`, `changed_before` filters.

Reply:
I am not against but that would be rather for a general filter functionality rework.